stylelint-plugin/rules/use-proper-token/index.test.ts

```diff
@@ -0,0 +1,103 @@
+import { describe, it, expect } from 'vitest'
+import getInappropriateTokens from './getInappropriateTokens'
+
+describe('getInappropriateTokens', () => {
+  it('returns empty array when no token used in value', () => {
+    const decl = {
+      prop: 'color',
+      value: 'red',
+    }
+
+    expect(getInappropriateTokens(decl)).toEqual([])
+  })
+
+  it('returns empty array when no `kui-` token used in value (CSS variable)', () => {
```

Comment:
Since you're going to have two separate sets of tests `CSS Custom Properites` and `SCSS variables` can you add two child `describe` blocks within the root to separate the tests? 

```ts
describe('CSS Custom Properties, () => {})
describe('SCSS variables, () => {})
```